apps/modernization-ui/src/pages/addPatient/AddPatient.tsx

```diff
@@ -221,13 +221,13 @@ export default function AddPatient() {
                                     <div className="width-full text-bold flex-row display-flex flex-align-center flex-justify">
                                         New patient
                                         <div className="button-group">
-                                            {!isValid && (
+                                            {isValid && (
```

Comment:
The `isValid` flag is coming from form validation, this change would allow adding a new Patient with invalid values.  The real issue is there is no definition of what "missing values" means.